include/tscpp/util/YamlCfg.h

```diff
@@ -81,3 +82,9 @@ namespace Yaml
 
 } // end namespace Yaml
 } // end namespace ts
+
+namespace swoc
+{
+// This needs to be in namespace "swoc" or "YAML" or ADL doesn't find the overload.
+extern BufferWriter &bwformat(BufferWriter &w, bwf::Spec const &spec, YAML::Mark const &mark);
```

Comment:
Why `extern`? Won't this have external linkage by default?

Reply:
Habit.